src/components/radioButton/demoBasicUsage/index.html

```diff
@@ -4,9 +4,9 @@
 
     <md-radio-group ng-model="data.group1">
 
-      <md-radio-button value="Apple" aria-label="Label 1">Apple</md-radio-button>
```

Comment:
These label attributes aren't necessary when the item has child text, because the utility dynamically populates `aria-label`.